### DISCUSSION TITLE
interface sfp duplex towards 2 front ports

### DISCUSSION THREAD

**Discussion by snorte** [netbox-community/netbox#7289] [Q&A]
There has to be some solution to get a standard sfp interface with a duplex singelmode fiber connection to two front port ODF...?

**Comment by candlerb** [comment]
The best you can do currently is to run a single cable from the interface to the first port of the ODF, and mark the adjacent port of the ODF as "connected".

![image](https://user-images.githubusercontent.com/44789/133776436-2fbe8357-7954-4be9-ac0a-4fa6d38ace63.png)

Alternatively, if your entire fibre infrastructure is duplex, then you can model each *pair* of LC ports as a single frontport.
